CHANGELOG.md

```diff
@@ -6,6 +6,35 @@
 
 ### Changed
 
+- CRD manifest `spec.version` is still supported, but users will see a warning message if `spec.versions` is not present and an error if `spec.versions` is populated but the version in `spec.version` is not in `spec.versions`.
+
+### Breaking changes
+
+- Upgrade Kubernetes version from `kubernetes-1.13.4` to `kubernetes-1.14.1`
+- Upgrade `github.com/operator-framework/operator-lifecycle-manager` version from `b8a4faf68e36feb6d99a6aec623b405e587b17b1` to `0.10.1`
+- Upgrade [`controller-runtime`](https://github.com/kubernetes-sigs/controller-runtime) version from `v0.1.12` to `v0.2.0-beta.3`
+  - The package `sigs.k8s.io/controller-runtime/pkg/runtime/scheme` is deprecated, and contains no code. Replace this import with `sigs.k8s.io/controller-runtime/pkg/scheme` where relevant.
+  - The package `sigs.k8s.io/controller-runtime/pkg/runtime/log` is deprecated. Replace this import with `sigs.k8s.io/controller-runtime/pkg/log` where relevant.
+  - The package `sigs.k8s.io/controller-runtime/pkg/runtime/signals` is deprecated. Replace this import with `sigs.k8s.io/controller-runtime/pkg/manager/signals` where relevant.
+  - [`sigs.k8s.io/controller-runtime/pkg/client.Client`](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/client/interfaces.go#L61)'s `List()` method has been updated: `List(ctx context.Context, opts *client.ListOptions, list runtime.Object) error` is now `List(ctx context.Context, list runtime.Object, opts ...client.ListOptionFunc) error`. To migrate:
+      ```go
+      import (
+        "context"
+
+        "sigs.k8s.io/controller-runtime/pkg/client"
+      )
+
+      ...
+
+      listOpts := &client.ListOptions{}
+      // Old
+      err = r.client.List(context.TODO(), listOps, podList)
+      // New
+      err = r.client.List(context.TODO(), podList, client.UseListOptions(listOps))
+      ```
+- CRD file names were previously of the form `<group>_<version>_<kind>_crd.yaml`. Now that CRD manifest `spec.version` is deprecated in favor of `spec.versions`, i.e. multiple versions can be specified in one CRD, CRD file names have the form `<full group>_<resource>.yaml`. `<resource>` is the plural lower-case CRD Kind found at `spec.names.plural`.
```

Comment:
Per offline discussion we'll keep the `_crd` suffix to keep the naming congruent with the CR manifest: `<full group>_<resource>_crd.yaml`